classes/requests/helpers/class-nets-easy-checkout-helper.php

```diff
@@ -44,6 +44,9 @@ public static function get_checkout( $checkout_flow = 'embedded', $order_id = nu
 			if ( 'all' !== get_option( 'woocommerce_allowed_countries' ) ) {
 				$checkout['shipping']['countries'] = self::get_shipping_countries();
 			}
+			if ( is_user_logged_in() ) {
+				$checkout['consumer'] = self::get_em_customer_data();
+			}
```

Comment:
* I don't think we the if statement need is_user_logged_in(). This means that a logged out customer that has entered a postal number will never be sent to Nets.
* The helper function for redirect flow is called get_consumer_address. I think the naming should be quite similar on these two functions, but give information if it is related to order or checkout/embedded.